tests/management/group/test_view.py

```diff
@@ -176,6 +176,21 @@ def test_update_group_success(self, mock_request):
         self.assertIsNotNone(response.data.get('uuid'))
         self.assertEqual(updated_name, response.data.get('name'))
 
+    @patch('management.principal.proxy.PrincipalProxy.request_filtered_principals',
+           return_value={'status_code': 200, 'data': []})
+    def test_update_platform_default_group_success(self, mock_request):
+        """Test that updating a default group changes system flag."""
+        group = Group.objects.filter(name='groupDef').first()
+        updated_name = group.name + '_update'
+        test_data = {'name': updated_name}
+        url = reverse('group-detail', kwargs={'uuid': group.uuid})
+        client = APIClient()
+        response = client.put(url, test_data, format='json', **self.headers)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        self.assertIsNotNone(response.data.get('uuid'))
+        self.assertEqual(updated_name, response.data.get('name'))
```

Comment:
Should we have an assertion here to check the `system` flag value?

Reply:
Good idea, adding that